alfalfa_worker/jobs/openstudio/step_run.py

```diff
@@ -73,6 +73,9 @@ def check_simulation_stop_conditions(self) -> bool:
             return True
         if not self.ep.is_running:
             return True
+        if self.get_energyplus_datetime() >= self.end_datetime:
```

Comment:
I think we should make this call a method similar to `modelica/step_run.py`. That way future model runs will implement something similar.